api/cpp/include/slint_platform.h

```diff
@@ -157,6 +157,49 @@ class WindowAdapter
     }
 };
 
+/// An event that is passed to the Platform::invoke_from_event_loop function and needs to be invoked
+/// in the event loop.
+class PlatformEvent
```

Comment:
An alternative is to move this class within the Platform class,  as `Platform::PlatformEvent`  or `Platform::Event`

Reply:
I like the `PlatformEvent` in this patch.